examples/main.tf

```diff
@@ -0,0 +1,137 @@
+
+provider "aws" {
```

Comment:
we dont create providers in main.tf or any other tf that are checked in. you should have it locally in a file named provider.tf that is already git ignored. While this module runs in pipeline, the provider is created by the pipeline